ui/eureka/eureka.css

```diff
@@ -251,6 +252,15 @@
   border-bottom: 2px dashed rgba(255, 255, 255, 0.3);
 }
 
+#portal-timer-container {
```

Comment:
What do you think about combining these with weather (like on top of the current weather?) just to avoid having too many boxes spread about?

Reply:
I decided on creating a separate box for it because the weather container is already pretty tall compared to the Hydatos map  and I wanted to keep it as compact as possible.
The layout of the weather box is also pretty fixed, there's always the time of day and then 5 weathers visible, whereas the portal timer is only going to be popping up when it's relevant to be shown. I think that having the box resize and an extra element appearing/disappearing would be pretty distracting personally, but I'm open to others' thoughts on this.

Reply:
Ok, that seems reasonable!